tooling/cli/schema.json

```diff
@@ -2606,12 +2606,12 @@
             }
           ]
         },
-        "dangerousExternalCommandAccess": {
-          "description": "Allow external urls to send command to Tauri.\n\nBy default, external urls do not have access to `window.__TAURI__`, which means they cannot communicate with the commands defined in Rust. This prevents attacks where an externally loaded malicious or compromised sites could start executing commands on the user's device.\n\nThis configuration allows a set of external urls to have access to the Tauri commands. Wildcards patterns are accepted and can be used to allow either all paths or a subset of paths for a domain.\n\n**WARNING:** Only use this option if you either have internal checks against malicious external sites or you can trust the allowed external sites. You application might be vulnerable to dangerous Tauri command related attacks otherwise.",
+        "dangerousRemoteUrlIpcAccess": {
+          "description": "Allow external domains to send command to Tauri.\n\nBy default, external domains do not have access to `window.__TAURI__`, which means they cannot communicate with the commands defined in Rust. This prevents attacks where an externally loaded malicious or compromised sites could start executing commands on the user's device.\n\nThis configuration allows a set of external domains to have access to the Tauri commands. Wildcards patterns are accepted and can be used to allow either all paths or a subset of paths for a domain.\n\n**WARNING:** Only use this option if you either have internal checks against malicious external sites or you can trust the allowed external sites. You application might be vulnerable to dangerous Tauri command related attacks otherwise.",
```

Comment:
Wildcards are not longer a thing by default, so we should remove this documentation part.
I would also add that the developer must ensure trust of the allowed domains during the lifetime of the desktop application as subdomain takeover is a huge risk.